File: AmongNeewAmigosBackEnd/src/main/java/leandromangieri/portfolio/core/dto/HomeDetailsDto.java
package leandromangieri.portfolio.core.dto;

import lombok.Data;

import java.util.Map;

@Data
public class HomeDetailsDto {
    private int pollControlStatus;
    private String voteTitle;
    private Map<Integer,String> voteOptions;
    private boolean hasUserVoted;
    private String userVote;

    private String titleLastPoll;
    private Map<String, Integer> resultsLastPoll;
    private int totalVotes;
}

File: AmongNeewAmigosFrontEnd/AmongNeewAmigosFrontEnd/src/app/pages/dtos/HomeDetailsDto.ts
export class HomeDetailsDto {
    constructor() {
    }
    pollControlStatus: number;
    voteTitle: string;
    voteOptions: Map<number, string>;
    hasUserVoted: boolean;
    userVote: string;

    titleLastPoll: string;
    resultsLastPoll: Map<string,number>;
    totalVotes: number;

}